sentinel-core/src/test/java/com/alibaba/csp/sentinel/slots/statistic/metric/OccupiableBucketLeapArrayTest.java

```diff
@@ -18,13 +20,16 @@
  * @author jialiang.linjl
  */
 public class OccupiableBucketLeapArrayTest {
```

Comment:
I added `@Retry` to all test cases of this class since I observed flaky failures in travis build history. I assume this is a common issue of all test cases in this class.